### DISCUSSION TITLE
Open 3dm file.

### DISCUSSION THREAD

**Discussion by taniaivanova** [hjoykim/THREE#34] [General]
Is there any example of how to open and view a 3dm file?

**Comment by hjoykim** [comment]
Thank you for your interest.

The 3dm file loader has not been implemented yet.
I'll write it soon and commit it with an example.

**Reply by taniaivanova** [reply]
OK, I will be very grateful to you ... The more types of 3D files it can read, it would be great.

**Reply by hjoykim** [reply]
I've added 3dm loader and example.
if you have any problem, please let me know.

**Comment by hjoykim** [comment]
hello,

I added AssimpLoader,
